corehq/apps/style/templates/style/includes/analytics_all.html

```diff
@@ -3,13 +3,10 @@
 {% load compress %}
 {% load hq_shared_tags %}
 
-<!DOCTYPE html>
-    {% include 'style/includes/analytics_google.html' %}
-    {% include 'style/includes/analytics_pingdom.html' %}
-    {% include 'style/includes/analytics_kissmetrics.html' %}
-    {% include 'style/includes/analytics_hubspot.html' %}
-</html>
```

Comment:
Sorry, this might have been me. I think I refactored some things without paying attention to these tags. Thanks for the cleanup